pkg/system/rm_unix.go

```diff
@@ -30,7 +30,7 @@ func EnsureRemoveAll(dir string) error {
 	exitOnErr := make(map[string]int)
 	maxRetry := 50
 
-	// Attempt to unmount anything beneath this dir first
+	// Attempt to unmount anything beneath this dir first. This is a no-op on Windows
```

Comment:
this is `rm_unix.go`, I think it makes no sense to reference Windows here.

Reply:
Ah 🤦 yes, your PR renamed it to a `_unix.go`. I'll have to change

Reply:
Removed